config/initializers/hesa_degree_types.rb

```diff
@@ -1,92 +1,86 @@
-  [210, "Master of Social Studies (M SS)"],
-  [211, "Master of Engineering (M ENG)"],
-  [212, "Master of Law (M LAW)"],
-  [213, "Master of Business Administration (MBA)"],
-  [214, "Master of Chemistry (M CHEM)"],
-  [215, "Master of Physics (M PHYS)"],
-  [300, "Doctor of Divinity (DD)"],
-  [301, "Doctor of Civil Law (DCL)"],
-  [302, "Doctor of Medicine (MD)"],
-  [303, "Doctor of Music (D MU)"],
-  [304, "Doctor of Science (D SC)"],
-  [305, "Doctor of Philosophy (D PHIL)"],
-  [306, "PhD (PHD)"],
-  [400, "First Degree"],
-  [401, "Higher Degree"],
-  [402, "Degree equivalent (inc. foreign qualifications)"],
-  [999, "Unknown"],
+  [1, nil, 'BEd', :bachelor],
+  [3, nil, 'BSc/Education', :bachelor],
+  [5, nil, 'BTech/Education', :bachelor],
+  [7, nil, 'BA/Education', :bachelor],
+  [9, nil, 'BA Combined Studies/Education of the Deaf', :bachelor],
+  [12, nil, 'BA with intercalated PGCE', :bachelor],
+  [51, 'BA', 'Bachelor of Arts', :bachelor],
+  [52, 'BA ECON', 'Bachelor of Arts Economics', :bachelor],
+  [53, 'BA ARCH', 'Bachelor of Arts in Architecture', :bachelor],
+  [54, 'BAO', 'Bachelor of the Art of Obstetrics', :bachelor],
+  [55, 'BA ARCH', 'Bachelor of Architecture', :bachelor],
+  [56, 'B ASC', 'Bachelor of Applied Science', :bachelor],
+  [57, 'B AG', 'Bachelor of Agriculture', :bachelor],
+  [58, 'B ACC', 'Bachelor of Accountancy', :bachelor],
+  [59, 'B ADMIN', 'Bachelor of Administration', :bachelor],
+  [60, 'BBA', 'Bachelor of Business Administration', :bachelor],
+  [61, 'BCS', 'Bachelor of Combined Studies', :bachelor],
+  [62, 'B COMM', 'Bachelor of Commerce', :bachelor],
+  [63, 'BD', 'Bachelor of Divinity', :bachelor],
+  [64, 'BDS', 'Bachelor of Dental Surgery', :bachelor],
+  [65, 'B ENG', 'Bachelor of Engineering', :bachelor],
+  [66, 'B ENG/BS', 'Bachelor of Engineering with Business Studies', :bachelor],
+  [67, 'BFA', 'Bachelor of Fine Art', :bachelor],
+  [68, 'BGS', 'Bachelor of General Studies', :bachelor],
+  [69, 'BH', 'Bachelor of Humanities', :bachelor],
+  [70, 'B HYG', 'Bachelor of Hygiene', :bachelor],
+  [71, 'LLB', 'Bachelor of Law', :bachelor],
+  [72, 'B LIB', 'Bachelor of Librarianship', :bachelor],
+  [73, 'BLS', 'Bachelor of Librarianship and Info. Studies', :bachelor],
+  [74, 'B LITT', 'Bachelor of Literature', :bachelor],
+  [75, 'BL ECON', 'Bachelor of Land Economy', :bachelor],
+  [76, 'B MED SC', 'Bachelor of Medical Science', :bachelor],
+  [77, 'BM', 'Bachelor of Medicine', :bachelor],
+  [78, 'B MET', 'Bachelor of Metallurgy', :bachelor],
+  [79, 'B MET/EN', 'Bachelor of Metallurgy and Engineering', :bachelor],
+  [80, 'B MU', 'Bachelor of Music', :bachelor],
+  [81, 'BN', 'Bachelor of Nursing', :bachelor],
+  [82, 'B PHARM', 'Bachelor of Pharmacy', :bachelor],
+  [83, 'B SC', 'Bachelor of Science', :bachelor],
+  [84, 'BSC/ECO', 'Bachelor of Science Economics', :bachelor],
+  [85, 'BSC/ENG', 'Bachelor of Science &amp; Engineering', :bachelor],
```

Comment:
Do we need to escape ampersands here? (we're quite far from the view)

Reply:
We don't, fixed!